Packs/AutofocusFeed/TestPlaybooks/playbook-AutofocusFeedTest.yml

```diff
@@ -0,0 +1,129 @@
+id: playbook-AutofocusFeedTest
```

Comment:
add a screenshot to the PR please, i want to see what you are testing

Reply:
will do - but it is just running the get-indicators command.